src/openforms/js/components/admin/forms/ModalJsonSchemaGeneration.js

```diff
@@ -0,0 +1,76 @@
+import {JSONEditor} from '@open-formulieren/monaco-json-editor';
+import PropTypes from 'prop-types';
+import {React, useContext, useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useGlobalState} from 'state-pool';
+
+import {FormContext} from 'components/admin/form_design/Context';
+import {FORM_ENDPOINT} from 'components/admin/form_design/constants';
+import Modal from 'components/admin/modals/Modal';
+import ErrorMessage from 'components/errors/ErrorMessage';
+import {get} from 'utils/fetch';
+import {currentTheme} from 'utils/theme';
+
+/**
+ * A container for JSON schema generation of a form.
+ */
+const ModalJsonSchemaGeneration = ({modalTitle, backendKey, modalSize = 'large'}) => {
+  const [modalOpen, setModalOpen] = useState(false);
+  const [hasError, setErrorState] = useState(false);
+  const [schema, setSchema] = useState(null);
+  const [theme] = useGlobalState(currentTheme);
+  const {form} = useContext(FormContext);
+
+  async function onShowModal() {
+    setModalOpen(true);
+    const params = new URLSearchParams({registration_backend_key: backendKey});
+    const response = await get(`${FORM_ENDPOINT}/${form.uuid}/json_schema?${params}`);
```

Comment:
https://github.com/open-formulieren/open-forms/blob/34c8023497c0bc49dab35a5e6729d4a6aff60e11/src/openforms/js/utils/fetch.js#L58 params support is baked in with an API similar to python:

```suggestion
    const response = await get(`${FORM_ENDPOINT}/${form.uuid}/json_schema`, {registration_backend_key: backendKey});
```